src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.Impl.cs

```diff
@@ -180,12 +180,14 @@ Semantics IView.Semantics
 		internal Semantics SetupSemantics() =>
 			_semantics ??= new Semantics();
 
-		static void ValidatePositive(double value, string name)
+		static double ValidatePositive(double value)
```

Comment:
This should probably be called `EnsurePositive` - it's not really a validation method anymore.